manifests/npp.pp

```diff
@@ -2,7 +2,7 @@
   $user    = 'Administrator',
   $default = true,
 )  {
-  require classroom::agent::chocolatey
+  require classroom::windows::chocolatey
```

Comment:
this won't work. That's now a private class :/

I've kludged a fix that will work for now and when the chocolatey module is fixed, we can do it the right way.